armi/operators/operator.py

```diff
@@ -65,16 +73,28 @@ class Operator:  # pylint: disable=too-many-public-methods
     cs : CaseSettings object
             Global settings that define the run.
 
+    cycleNames : list
+        The name of each cycle. Cycles without a name are `None`.
+
+    stepLengths : list
+        A two-tiered list, where primary indices correspond to cycle and
+        secondary indices correspond to the length of each intra-cycle step (in days).
+
     cycleLengths : list
         The duration of each individual cycle in a run (in days). This is the entire cycle,
         from startup to startup and includes outage time.
 
+    burnSteps : list
+        The number of sub-cycles in each cycle.
+
     availabilityFactors : list
```

Comment:
shouldn't this be a two-tiered list as well? or are we by definition saying that a cycle has a pre-determined availability that doesn't change? Might be worth adding to the docstring the reason this is not two-tiered

Reply:
Sure, I see that confusion. 

The assumption is that the availability factor is on a cycle-basis, not on a step-basis. So every cycle has its own availability factor, but that availability factor is applied to the cycle as a whole.

Added some clarification.